akvo/rsr/views/project.py

```diff
@@ -410,6 +410,8 @@ def set_update(request, project_id, edit_mode=False, form_class=ProjectUpdateFor
         raise PermissionDenied
 
     if request.method == 'POST':
+        if not allow_update:
```

Comment:
If a user is not allowed to update, I think we can raise a `PermissionDenied` irrespective of what the request method is?

Reply:
The template is using `allow_update` and doing some error handling. So, this is good.